pkg/functions/kube.go

```diff
@@ -1299,7 +1299,7 @@ func GenerateWorkflowServiceName(info *igrpc.BaseInfo) (string, string) {
 	// NOTE: fndef.Files can be safely excluded
 
 	var strs []string
-	strs = []string{fndef.Cmd, fndef.ID, fndef.Image, fmt.Sprintf("%v", fndef.Scale), fmt.Sprintf("%v", fndef.Size), fmt.Sprintf("%v", fndef.Type)}
+	strs = []string{fndef.Cmd, fndef.ID, fndef.Image, fmt.Sprintf("%v", fndef.Size), fmt.Sprintf("%v", fndef.Type)}
```

Comment:
I can't see a reason why scale needs to be here, do you? This logic impacts namespace and global level functions as well, but it seems weird to me that you'd have two otherwise unique functions defined where the only difference is the scale.

Reply:
It would only make a difference if its 0 vs 1+ but not 1 vs 1+. I think that was my thinking. But in real life that should not make a difference. But if you define identical functions with 0 or 1 scale they are different and if you create the scale 1 later it would basically ignore the scale of 0 because it already exists. If I understand that right looking at it that late.

Reply:
But are there any sensible reasons to have two identical global or namespace functions with different scales? If you had a 0 scale one and then later decided you also want a 1 scale one, wouldn't it make more sense to replace the existing one, rather than create a new one?

Unless maybe separating them is good for some other reasons. Like more fine grained traffic management maybe?